### ISSUE TITLE
Sass List After formatting

### ISSUE THREAD

**Issue by Cyclone77** [tierratelematics/code-beautifier#5]
**After formatting**
``` scss
$animal-data: (puma,
black,
default),
(sea-slug,
blue,
pointer),
(egret,
white,
move);
@each $animal,
$color,
$cursor in $animal-data {
    .#{$animal}-icon {
        background-image: url('/images/#{$animal}.png');
        border: 2px solid $color;
        cursor: $cursor;
    }
}
```

**Comment by mike7515** [comment]
Fixed in the last release 1.7.2

**Comment by Cyclone77** [comment]
I'm sorry, 1.7.2 Still support is not good.
